<launch>
 <!-- Navigation Program  -->
<<<<<<< HEAD
    <arg name="distance_laser" default="0.5" />
    <arg name="speed_factor" default="0.1" />
    <arg name="lateral_speed" default="0.1" />
    <arg name="forward_speed" default="0.1" />
=======
    <arg name="distance_laser" default="0.4" />
    <arg name="speed_factor" default="0.1" />
    <arg name="lateral_speed" default="0.1" />
    <arg name="forward_speed" default="0.1" />

>>>>>>> 28ddeb7ef946c213a299d2dc712ddb2aa8d185e0
  <node name="wall_follower" pkg="rubot_control" type="rubot_wall_follower_holonomic_opt.py" output="screen" >
    <param name="distance_laser" value="$(arg distance_laser)"/>
    <param name="speed_factor" value="$(arg speed_factor)"/>
    <param name="lateral_speed" value="$(arg lateral_speed)"/>
    <param name="forward_speed" value="$(arg forward_speed)"/>
  </node>
</launch>
